File: basic/src/main/java/com/yuanshenbin/basic/log/LogImpl.java
package com.yuanshenbin.basic.log;

import android.util.Log;

public class LogImpl  extends LogAbstract{
    @Override
    void i(String tag, String msg) {
        Log.i(tag,msg);
    }

    @Override
    void i(String msg) {
        Log.i(TAG,msg);
    }

    @Override
    void e(String tag, String msg) {
        Log.e(tag,msg);
    }

    @Override
    void e(String msg) {
        Log.e(TAG,msg);
    }

    @Override
    void d(String tag, String msg) {
        Log.d(tag,msg);
    }

    @Override
    void d(String msg) {
        Log.d(TAG,msg);
    }

    @Override
    void v(String tag, String msg) {
        Log.v(tag,msg);
    }

    @Override
    void v(String msg) {
        Log.v(TAG,msg);
    }

    @Override
    void w(String tag, String msg) {
        Log.w(tag,msg);
    }

    @Override
    void w(String msg) {
        Log.e(TAG,msg);
    }
}

File: basic-kotlin/src/main/java/com/yuanshenbin/basic/log/LogImpl.kt
package com.yuanshenbin.basic.log

import android.util.Log

/**
 * author : yuanshenbin
 * time   : 2023/3/9
 * desc   :
 */
open class LogImpl : LogAbstract() {
    override fun i(tag: String?, msg: String?) {
        Log.i(tag, msg!!)
    }

    override fun i(msg: String?) {
        Log.i(tag, msg!!)
    }

    override fun e(tag: String?, msg: String?) {
        Log.e(tag, msg!!)
    }

    override fun e(msg: String?) {
        Log.e(tag, msg!!)
    }

    override fun d(tag: String?, msg: String?) {
        Log.d(tag, msg!!)
    }

    override fun d(msg: String?) {
        Log.d(tag, msg!!)
    }

    override fun v(tag: String?, msg: String?) {
        Log.v(tag, msg!!)
    }

    override fun v(msg: String?) {
        Log.v(tag, msg!!)
    }

    override fun w(tag: String?, msg: String?) {
        Log.w(tag, msg!!)
    }

    override fun w(msg: String?) {
        Log.w(tag, msg!!)
    }
}